cinema_service/urls.py

```diff
@@ -5,4 +5,5 @@
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("__debug__/", include("debug_toolbar.urls")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
The URL pattern for the `user` app is correctly included with a namespace. Ensure that the `user.urls` module defines a `app_name` variable to support namespacing.